library/pcs_cluster.py

```diff
@@ -49,6 +49,10 @@
     required: false
     default: default
     choices: ['default', 'udp', 'udpu', 'knet']
+  transport_options:
+    description:
+      - additional options for transports (must not be used with pcs 0.9)
```

Comment:
(low priority) I would rephrase it to something like "available only with `pcs-0.10`"

Reply:
(medium priority) "this option can be used only when `transport` options is specified (non-default)"